httpapi/userapi/types.go

```diff
@@ -351,9 +351,10 @@ func toFundingAgreementListResponse(ctx context.Context,
 
 // MintNFTRequest holds required fields for minting NFT
 type MintNFTRequest struct {
-	DocumentID     byteutils.HexBytes `json:"document_id" swaggertype:"primitive,string"`
-	DepositAddress common.Address     `json:"deposit_address" swaggertype:"primitive,string"`
-	ProofFields    []string           `json:"proof_fields"`
+	DocumentID          byteutils.HexBytes    `json:"document_id" swaggertype:"primitive,string"`
+	DepositAddress      common.Address        `json:"deposit_address" swaggertype:"primitive,string"`
+	AssetManagerAddress byteutils.OptionalHex `json:"asset_manager_address" swaggertype:"primitive,string"`
```

Comment:
We can even read the property `assetManager ` from the NFT Registry passed (if it is public), to avoid maintaining it in the code. WDYT?

Reply:
This piece of code will be removed as soon as the bridge integration is done. Then we will pull the asset manager address as part of default config. I have added this here since it easier to delegate the address to API caller, which will be functional tests. I would rather have us wait until bridge is in